.dockerignore

```diff
@@ -3,11 +3,11 @@
 *
 
 # Include source directories and files required for building.
+!.git
```

Comment:
issue: This is likely not a good idea, as it will include the full git history in built docker images. If this is to enable VCS versioning, I would instead highly encourage making the build job extract the version from VCS and pass it as a build parameter to docker.

Reply:
The container build is multistage build. The actual Karapace container has only the copied virtual environment: https://github.com/Aiven-Open/karapace/blob/main/container/Dockerfile#L42

Reply:
Ah right, and `KARAPACE_VERSION` is already passed as a build parameter actually, perhaps that's not necessary any longer then.

Reply:
Mentioned in 1-1, just copying here for completeness too: including the `.git` directory, even in the build stage, will cause all layers of the image to have to be rebuilt if there are changes in that directory -> which means for instance doing a `git fetch` can cause a complete rebuild.

Reply:
Removed the requirement, added use of `SETUPTOOLS_SCM_PRETEND_VERSION` to `Dockerfile`.